### DISCUSSION TITLE
Added chip Component

### DISCUSSION THREAD

**Discussion by webvs2** [vueComponent/ant-design-vue#6098] [Ideas]
Used to convey small pieces of information. The close property makes the paper interactible，reference：https://vuetify.cn/zh-Hans/components/chips/#%E7%A4%BA%E4%BE%8B

**Comment by webvs2** [comment]
@tangjinzhou  How do you think?

**Comment by wsxe9988** [comment]
@tangjinzhou follow the latest design specifications of antd 4.x
